fusor-ember-cli/app/controllers/deployment-new.js

```diff
@@ -23,6 +24,16 @@ export default Ember.Controller.extend(DeploymentControllerMixin, DisableTabMixi
   hasLifecycleEnvironment: Ember.computed('model.lifecycle_environment', 'useDefaultOrgViewForEnv', function() {
     return (!!(this.get('model.lifecycle_environment.id')) || this.get('useDefaultOrgViewForEnv') || this.get('selectedEnvironmentDeploymentNew'));
   }),
-  hasNoLifecycleEnvironment: Ember.computed.not('hasLifecycleEnvironment')
+  hasNoLifecycleEnvironment: Ember.computed.not('hasLifecycleEnvironment'),
+
+  isValidCommonPassword: Ember.computed.alias("deploymentNewSatelliteIndex.isValidCommonPassword"),
+
+  isValidNameAndPassword: Ember.computed('isValidDeploymentName', 'isValidCommonPassword',
+    function() {
+      return (this.get('isValidDeploymentName') && this.get('isValidCommonPassword'));
+    }
+  ),
```

Comment:
this needs to be in `deployment-new` and `deployment` rather than the `disable-tab-mix`